tutorials/ui5-webcomponents-react-charts/ui5-webcomponents-react-charts.md

```diff
@@ -36,39 +36,60 @@ UI5 Web Components for React also comes with a chart library. In this tutorial,
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 2: ](Add charts to Card component)]
-1. Start with the `LineChart` and add it underneath the `Text` component.
+1. Start with the `LineChart`. You can add it underneath the `Text` component. Then pass the `dimensions` and `measures` prop with an empty array as value.
 
     ```JavaScript / JSX
     <Text style={spacing.sapUiContentPadding}>
       This is the content area of the Card
     </Text>
-    <LineChart />
+    <LineChart measures={[]} dimensions={[]} />
     ```
 
-    Well, that didn't change much, didn't it? It's because the chart didn't receive any data, and therefore the content is empty.
+    Now you should see a placeholder for the `LineChart` within the `Card` content. The reason for this is that the chart has not received any data and therefore the placeholder is displayed.
 
-2. Add data and corresponding labels to your component (right above the `return` statement).
+2. Add data to your component (right above the `return` statement).
 
     ```JavaScript / JSX
-    const datasets = [{
-        label: "Stock Price",
-        data: [65, 59, 80, 81, 56, 55, 40]
-    }];
-    const labels = [
-        "January",
-        "February",
-        "March",
-        "April",
-        "May",
-        "June",
-        "July"
+    const dataset = [
+      {
+        month: "January",
+        data: 65
+      },
+      {
+        month: "February",
+        data: 59
+      },
+      {
+        month: "March",
+        data: 80
+      },
+      {
+        month: "April",
+        data: 81
+      },
+      {
+        month: "May",
+        data: 56
+      },
+      {
+        month: "June",
+        data: 55
+      },
+      {
+        month: "July",
+        data: 40
+      }
     ];
     ```
 
-3. Now add `datasets` and `labels` to your `LineChart` components.
+3. Now add the `dataset` to your `LineChart` and configure the `dimensions` and `measures` props.
```

Comment:
```suggestion
3. Now add the `dataset` to your `LineChart` and configure the `dimensions` and `measures` properties.
```

Reply:
`Props` is a special keyword in [React](https://reactjs.org/docs/components-and-props.html), so in my opinion it should be referred to as such, otherwise it could confuse the user. What do you think?

Reply:
If it is a key word, then it is fine. I assumed that you used props as a abbreviation for properties.